### ISSUE TITLE
Enzyme dataset

### ISSUE THREAD

**Issue by MohsenFazaeli** [rusty1s/pytorch_geometric#2107]
## 🐛 Bug

Enzyme dataset contains 18 node features and assigns one of three labels to each node. However, it's not the case with PyTorch geometric it loads node labels as features and does not load the original 18-dimension features

# Benchmark data link 
http://graphkernels.cs.tu-dortmund.de/

# reproduce
import torch
#Install required packages.
!pip install -q torch-scatter==latest+cu101 -f https://pytorch-geometric.com/whl/torch-1.7.0.html
!pip install -q torch-sparse==latest+cu101 -f https://pytorch-geometric.com/whl/torch-1.7.0.html
!pip install -q git+https://github.com/rusty1s/pytorch_geometric.git

from torch_geometric.datasets import TUDataset
dataset = TUDataset(root='/tmp/ENZYMES', name='ENZYMES')

print(dataset)
print(len(dataset))
print(dataset.num_classes)
print(dataset.num_node_features)
g1 = dataset.get(0)
print(g1.y)
print(g1.num_node_features)
print(g1.x)
print(g1.num_nodes)

**Comment by rusty1s** [comment]
You need to use `use_node_attr=True` when loading the dataset.

**Comment by MohsenFazaeli** [comment]
Thanks for your advice @rusty1s. My task is node classification and I need to separate them to labels and features. I did the following code. However, subgraphs .x are still have 21 feature. I think It is possible to separate node features and labels for all subgraph at once but I don't know how. 
Could you please tell me how to do it?

num_node_attributes = dataset.num_node_attributes
dataset.data.ny = dataset.data.x[:, num_node_attributes:]
dataset.data.x = dataset.data.x[:, :num_node_attributes]
